ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/3scale_single_tenant.yml

```diff
@@ -0,0 +1,122 @@
+# The following is only appropriate when create generic tenants whose names are sequential
+# The following is skipped when creating a "named" tenant, ie; openbanking-dev  
+
+# Name of 3scale API administrator of the tenant
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_admin_id: "{{ ocp4_workload_lpe_3scale_tenant_admin_name_base }}"
+
+# Name of 3scale tenant organisation
+- set_fact:
+    ocp4_workload_lpe_3scale_org_name: "{{ocp4_workload_lpe_3scale_tenant_user_name_base}}-tenant"
+
+# Name of 3scale tenant namespace
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_gw_project: "{{ ocp4_workload_lpe_3scale_tenant_user_name_base }}-gw"
+
+# Name of 3scale tenant namespace display name
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_gw_projectDisplay: "{{ ocp4_workload_lpe_3scale_tenant_user_name_base }} APICast Gateways Project"
+
+# Set the tenant admin email
+- name: "{{ocp4_workload_lpe_3scale_org_name}}      1)  **********   TENANT CREATION  **********"
+  set_fact: 
+    ocp4_workload_lpe_3scale_tenant_admin_email: "{{ocp4_workload_lpe_3scale_admin_email_user}}@{{ocp4_workload_lpe_3scale_admin_email_domain}}"
+- debug: 
+    msg: "{{ocp4_workload_lpe_3scale_org_name}} ocp4_workload_lpe_3scale_tenant_admin_email = {{ocp4_workload_lpe_3scale_tenant_admin_email}}"
+
+# Wait for 3scale tenant to be created
+- name: "Create the {{ocp4_workload_lpe_3scale_org_name}} CRDs"
+  k8s:
```

Comment:
I would start using the collections. So instead of just `k8s` use `kubernetes.core.k8s`

Reply:
Resolved